server/sonar-server/src/main/java/org/sonar/server/issue/ws/TagsAction.java

```diff
@@ -38,7 +38,10 @@
 import org.sonar.db.organization.OrganizationDto;
 import org.sonar.server.issue.index.IssueIndex;
 import org.sonar.server.rule.index.RuleIndex;
+import org.sonar.server.ws.WsUtils;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.sonar.server.ws.WsUtils'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)